cdc/processor/sinkmanager/table_sink_worker.go

```diff
@@ -376,31 +376,14 @@ func (w *sinkWorker) handleTask(ctx context.Context, task *sinkTask) (err error)
 		)
 
 	} else {
-		// This happens when there is no workload for this table on upstream.
-		if lastTimeCommitTs == 0 {
-			lastTimeCommitTs = upperBound.CommitTs
-			log.Debug("No more events, set lastTimeCommitTs to upperBound",
-				zap.String("namespace", w.changefeedID.Namespace),
-				zap.String("changefeed", w.changefeedID.ID),
-				zap.Int64("tableID", task.tableID),
-				zap.Any("lowerBound", lowerBound),
-				zap.Any("upperBound", upperBound),
-				zap.Uint64("lastTimeCommitTs", lastTimeCommitTs),
-				zap.Bool("splitTxn", w.splitTxn),
-			)
-			err := advanceTableSinkAndResetCurrentSize(lastTimeCommitTs)
-			if err != nil {
-				return errors.Trace(err)
-			}
-			lastPos = upperBound
-		}
 		// This happens when:
 		// 1. We just leave the last txn in the events.
 		//    Because we only advance the table sink when meet a new commit ts.
 		// 2. We append all the events to the table sink, but because we do not reach the update interval,
 		//    we do not advance the table sink.
 		if len(events) > 0 || currentTotalSize > 0 {
-			if err := appendEventsAndRecordCurrentSize(currentCommitTs); err != nil {
+			lastTimeCommitTs = currentCommitTs
```

Comment:
Better to add comment about the update rule of `lastTimeCommitTs`, besides the variable name is not quite self explanatory

Reply:
Updated!